src/Debug/boneWeightShader.ts

```diff
@@ -0,0 +1,80 @@
+import { Nullable } from '../types';
+import { Skeleton, Bone } from '../Bones';
+import { AbstractMesh } from '../Meshes';
+import { DynamicTexture } from '../Materials';
+import { Scene } from '../scene';
+import { Color3 } from '../Maths';
+
+
+interface IBoneWeightOptions{
+    name: Nullable<string>
+    skeleton: Skeleton
+    mesh: AbstractMesh    
+    renderingMode: number
+
+    //0-1 number, then a color3 in duplet for Selected node weights
+    selectedColorKnots : (number|Color3)[]
+    //0-1 number, then a color3 in duplet for skeleton color map
+    mapColorKnots : (number|Color3)[]
+}
+
+/*interface IBoneWeightShaderGradientCache{
+    selectedWeight: Nullable<number[]>|DynamicTexture
+    colorMap: Nullable<number[]>|DynamicTexture
+}*/
+
+export class BoneWeightShader{    
+    public name?: Nullable<string>    
+    public skeleton: Skeleton
+    public mesh: AbstractMesh
+    private _scene: Scene  
+    //private _renderingMode: number

```

Comment:
need to be removed

Reply:
What needs to be removed?

Reply:
Ohh I get you!  Yeah, for sure Ill have all comments stripped here soon.

My phone was weird and was showing the entire commit and I got confused.